-------------------------------------------------------------------------------
Test set: gapchenko.llttz.test.ListInsertTest
-------------------------------------------------------------------------------
Tests run: 2, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 0 sec <<< FAILURE! - in gapchenko.llttz.test.ListInsertTest
testListInsert1(gapchenko.llttz.test.ListInsertTest)  Time elapsed: 0 sec  <<< FAILURE!
java.lang.AssertionError: Should throw duplicate error
	at gapchenko.llttz.test.ListInsertTest.testListInsert1(ListInsertTest.java:19)

